apps/frontend/src/app/3_organisms/FundingHistoryFrame/FundingHistoryFrame.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { nanoid } from 'nanoid';
+import { useTranslation } from 'react-i18next';
+
+import {
+  NotificationType,
+  OrderDirection,
+  OrderOptions,
+  Pagination,
+  Table,
+  applyDataAttr,
+} from '@sovryn/ui';
+
+import { ExportCSV } from '../../2_molecules/ExportCSV/ExportCSV';
+import { useNotificationContext } from '../../../contexts/NotificationContext';
+import { useAccount } from '../../../hooks/useAccount';
+import { useBlockNumber } from '../../../hooks/useBlockNumber';
+import { translations } from '../../../locales/i18n';
+import { rskClient } from '../../../utils/clients';
+import {
+  DEFAULT_HISTORY_FRAME_PAGE_SIZE,
+  EXPORT_RECORD_LIMIT,
+} from '../../../utils/constants';
+import {
+  BitcoinTransfer,
+  BitcoinTransferDirection,
+  BitcoinTransfer_OrderBy,
+  useGetFundingLazyQuery,
+} from '../../../utils/graphql/rsk/generated';
+import { useGetFundingHistory } from './hooks/useGetFundingHistory';
+import { FundingHistoryType } from './types';
+import { columnsConfig, generateRowTitle } from './utils';
+
+const pageSize = DEFAULT_HISTORY_FRAME_PAGE_SIZE;
+
+export const FundingHistoryFrame: FC = () => {
+  const { t } = useTranslation();
+  const { account } = useAccount();
+  const { addNotification } = useNotificationContext();
+  const [fundingHistory, setFundingHistory] = useState<FundingHistoryType[]>(
+    [],
+  );
+  const { value: block } = useBlockNumber();
+
+  const [page, setPage] = useState(0);
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'updatedAtTimestamp',
+    orderDirection: OrderDirection.Desc,
+  });
+
+  const { data, loading, refetch } = useGetFundingHistory(
```

Comment:
This is returning both incoming and outgoing transactions, please make sure that they are labelled correctly and explorer links work for both directions - currently everything shows as incoming deposits

Reply:
[https://prnt.sc/S-CNrXmZlgv2](https://prnt.sc/S-CNrXmZlgv2) This is what I see for the address that you mentioned here [https://sovryn.atlassian.net/browse/SOV-1136?focusedCommentId=14277](https://sovryn.atlassian.net/browse/SOV-1136?focusedCommentId=14277)

Reply:
@pietro-maximoff it was still missing any checks for tx type :D `item.direction` is the prop from data
![image](https://user-images.githubusercontent.com/765825/219966095-5fd8a3ad-4239-4b0a-83c4-860da916aba0.png)

I created PR to get everything working - review and we'll merge it to your branch:
https://github.com/DistributedCollective/sovryn-dapp/pull/230

Noticed an issue with the data returned from Graph for BTC txid's prefixing them with "0x", so I added a hacky `substring()` fix for it in my changes, we'll remove it once Graph data is fixed.